models/projects.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
         },
         status : {
             type : String,
-            default : "notStarted",
+            default : "started",
```

Comment:
@priyanka-TL we should record completedDate as well, in migration set the updatedAt os completedDate